src/main/java/it/gov/pagopa/payhub/activities/service/classifications/trclassifiers/RtIufTesClassifier.java

```diff
@@ -12,10 +12,10 @@
 public class RtIufTesClassifier implements TransferClassifier {
 
 	@Override
-	public ClassificationsEnum classify(TransferDTO transferDTO, PaymentsReportingDTO paymentsReportingDTO, TreasuryDTO treasuryDTO) {
+	public ClassificationsEnum classify(TransferDTO transferDTO, PaymentsReportingDTO paymentsReportingDTO, Treasury treasuryDTO) {
 		if(transferDTO != null && paymentsReportingDTO != null && treasuryDTO != null &&
 			getAmountCents(transferDTO).equals(getAmountCents(paymentsReportingDTO)) &&
-			getAmountCents(transferDTO).equals(getAmountCents(treasuryDTO))) {
+			getAmountCents(transferDTO).equals(treasuryDTO.getBillAmountCents())) {
```

Comment:
the amount field retrieve logic should still be centralized as done per other entities
```suggestion
			getAmountCents(transferDTO).equals(getAmountCents(treasuryDTO))) {
```